sklearn/_fast_gradient_boosting/loss.py

```diff
@@ -0,0 +1,238 @@
+"""
+This module contains the loss classes.
+
+Specific losses are used for regression, binary classification or multiclass
+classification.
+"""
+# Author: Nicolas Hug
+
+from abc import ABC, abstractmethod
+
+import numpy as np
+from scipy.special import expit
+try:
+    from scipy.special import logsumexp
+except ImportError:
```

Comment:
maybe add a comment about after which version of scipy we can remove the `ImportError` part?